spec/models/tasuku/taskables/taskable_spec.rb

```diff
@@ -44,6 +45,25 @@ def answers
           expect(taskable).to be_completed_by user
         end
       end
+
+      context 'with a incorrectly answered question task' do
+        before { create :question_answer, author: user, options: [taskable_question.options.first] }
+        ::Tasuku.temporarily(update_answers: true) do
```

Comment:
Actually this stuff is wrong i learned yesterday. I need to put the temporary config around the create action above, not on the expect. I have updated this on a new branch I'm creating for quiz. We might just let this branch be untill i make a PR for the Quiz thing..